resolwe/storage/connectors/baseconnector.py

```diff
@@ -161,15 +161,16 @@ def open_stream(self, url: "PathLike[str]", mode: str) -> Optional[BinaryIO]:
         """
         raise NotImplementedError
 
-    
-    def delete(self, urls: List[Union[str, PathLike]]):
+    def delete(self, url: str, urls: List[Union[str, PathLike]]):
         """Remove objects.
 
-        Since delete is potentially harmfull there are some sanity checks
-        in the main class. I suggest using them.
+        Since delete is potentially harmfull use validate_urls and validate_url
+        decorators on the implementation.
 
-        :param urls: URLs of the objects to delete. They must be relative else
-            ValueError is raised.
+        :param urls: URLs of the objects to delete. They must be relative with
+        respect to url argument.
+
+        :param url: base URL of the objets to delete.
```

Comment:
It would be nice to have the documentation in the same order as arguments.

Reply:
OK.